canonical/cos-lite-bundle
# tracing overlay

added tracing overlay file

- [ ] do we need to add anything to the readme/documentation?
- [x] test the overlay

## Testing instructions
Deploy the testing and tracing overlay:

```
tox -e render-edge
juju deploy ./bundle.yaml --trust \
  --overlay overlays/tracing-overlay.yaml \
  --overlay overlays/testing-overlay.yaml
```

Then, sync latest v2 tracing lib
```
jhack sync "*" --source-dirs ./lib/charms/tempo_k8s/v2/
```
Check grafana dashboard that tempo source has charm traces from (alertmanager, prometheus, loki, grafana, tempo, traefik, catalogue)

Add tls overlay:
```
juju deploy ./bundle.yaml --trust \
  --overlay overlays/tls-overlay.yaml \
  --overlay overlays/testing-overlay.yaml
```

verify you can still see the new charm traces for all apps (and grafana workload)
